docs/source/guide/stats.md

```diff
@@ -22,101 +24,102 @@ Task agreement shows the consensus between multiple annotators when labeling the
 
 You can also see how the annotations from a specific annotator compare to the prediction scores for a task, or how they compare to the ground truth labels for a task.
 
-For more about viewing agreement in Label Studio Enterprise, see [Verify model and annotator performance](quality.html#Verify-model-and-annotator-performance)
+For more about viewing agreement in Label Studio Enterprise, see [Verify model and annotator performance](quality.html#Verify-model-and-annotator-performance).
+
+## Agreement method
+
+The agreement method defines how [matching scores](stats.html#Matching-score) across all annotations for a task are combined to form a single inter-annotator agreement score.
+
+There are several possible methods you can specify on the **Annotation Settings** for a project.
+
+### Complete linkage
```

Comment:
@niklub lost in LSE 2.0
Where should it be used?

Reply:
These are matching score aggregation methods. We left only with "No grouping" due to complexity reasons. Not needed anymore in doc

Reply:
Perfect, I updated accordingly